nats-top.go

```diff
@@ -36,12 +36,15 @@ var (
 	caCertOpt     = flag.String("cacert", "", "Root CA cert")
 	skipVerifyOpt = flag.Bool("k", false, "Skip verifying server certificate")
 
+	// Nats context options
+	contextName = flag.String("c", "", "The name of the context to use. If not specified, the currently selected context is used.")
+
 	version = "0.0.0"
 )
 
 const usageHelp = `
 usage: nats-top [-s server] [-m http_port] [-ms https_port] [-n num_connections] [-d delay_secs] [-r max] [-o FILE] [-l DELIMITER] [-sort by]
-                [-cert FILE] [-key FILE] [-cacert FILE] [-k] [-b] [-v|--version] [-u|--display-subscriptions-column]
+                [-cert FILE] [-key FILE] [-cacert FILE] [-k] [-b] [-v|--version] [-u|--display-subscriptions-column] [-c context_name]
```

Comment:
same here, let's just use longer option for now
```suggestion
                [-cert FILE] [-key FILE] [-cacert FILE] [-k] [-b] [-v|--version] [-u|--display-subscriptions-column] [--context context_name]
```